### PR TITLE
add feat admin

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/74382
https://edu-redmine.sun-asterisk.vn/issues/74343
## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/views/cart/index.hbs [hunk 1]
```diff
@@ -52,9 +55,12 @@
                 {{/each}}
             </tbody>
         </table>
+        {{else}}
+<img src="https://mir-s3-cdn-cf.behance.net/project_modules/max_3840/4e9249104375869.5f61b55c5ebf5.jpg" alt="empty-cart" class="w-100 h-75">
+{{/if}}
```
Comment (huongnt-2545) [inline src/views/cart/index.hbs]: check indentation

File: src/public/js/product/product-detail.js [hunk 2]
```diff
@@ -28,9 +28,17 @@ if (btntoppings) {
...
 }
 
-const addToCart = (e, id) => {
+const addToCart = (e, id, userId) => {
   //convert size to boolean
-
+  if (!userId) {
+    swal.fire({
+      title: 'Error!',
+      text: 'You need to login to add to cart',
```
Comment (huongnt-2545) [inline src/public/js/product/product-detail.js]: E tìm hiểu I18n js nhá

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**